lib/src/async/web_driver.dart

```diff
@@ -37,10 +37,6 @@ class WebDriver implements SearchContext {
       {this.filterStackTraces: true})
       : this._prefix = uri.resolve('session/$id/');
 
-  /// Deprecated in favor of addEventListener.
-  @deprecated
-  Stream<WebDriverCommandEvent> get onCommand => _onCommandController.stream;
```

Comment:
You can completely remove `_onCommandController`, too – I'm pretty sure.

Reply:
Another nit – check the analyzer...

`StreamController` is shown, but not used...